### ISSUE TITLE
TS and VS CRD don't detect the pool members. Ingress works fine 

### ISSUE THREAD

**Issue by skenderidis** [F5Networks/k8s-bigip-ctlr#2233]
#### Setup Details
 CIS Version : 2.7.1
 Build: f5networks/k8s-bigip-ctlr:latest       
 BIGIP Version: Big IP 15.1
 AS3 Version: 3.33
 Agent Mode: AS3/CCCL       
 Orchestration: K8S/OSCP      
 Orchestration Version:       
 Pool Mode: Cluster/Nodeport      
 Additional Setup details: <Platform/CNI Plugins/ cluster nodes/ etc>

#### Description  
Deployed Grafana and both TS and VS CRD cannot detect any pool member.
When deployed CIS as ingress and configured an ingress resource, CIS detected the pool member

#### Steps To Reproduce
Deploy Grafana
git clone https://github.com/prometheus-operator/kube-prometheus
cd kube-prometheus
kubectl create -f manifests/setup
kubectl create -f manifests/

Apply TS CRD

```
apiVersion: "cis.f5.com/v1"
kind: TransportServer
metadata:
  name: grafana
  namespace: monitoring
  labels:
    f5cr: "true"
spec:
  virtualServerAddress: "192.168.3.48"
  virtualServerPort: 3000
  virtualServerName: grafana
  mode: standard
  snat: auto   
  pool:
    service: grafana
    servicePort: 3000
```

#### Expected Result 
The pool will show as 0 members

**Comment by trinaths** [comment]
@skenderidis - Please share the CIS config

**Comment by trinaths** [comment]
@skenderidis also working Ingress manifest.

**Comment by trinaths** [comment]
Created [CONTCNTR-3159] for internal tracking.

**Comment by skenderidis** [comment]
This has not been solved!! with 2.8.0 I still have the same problem

**Comment by trinaths** [comment]
@skenderidis is this issue resolved for you ?
